cmd/cloud.go

```diff
@@ -51,19 +53,24 @@ const (
 )
 
 //nolint:gochecknoglobals
-var exitOnRunning = os.Getenv("K6_EXIT_ON_RUNNING") != ""
+var (
+	exitOnRunning = os.Getenv("K6_EXIT_ON_RUNNING") != ""
+	showCloudLogs = os.Getenv("K6_SHOW_CLOUD_LOGS") != "false"
```

Comment:
A small wrapper around `strconv.ParseBool` would probably be better? I know it's a PITA to spend time on configuration in its current form, but no need to make it worse than it needs to be

Reply:
the problem is that .. I will need to move this inside the command as well

Reply:
erm that's what I meant by a "small wrapper" :sweat_smile: It can be something as simple as `var showCloudLogs = getEnvBool("K6_SHOW_CLOUD_LOGS")`

Reply:
see 7c6e60e